nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/main/java/org/apache/nifi/controller/repository/StandardProcessSession.java

```diff
@@ -175,6 +184,10 @@ public StandardProcessSession(final ProcessContext context) {
         this.sessionId = idGenerator.getAndIncrement();
         this.connectableDescription = description;
 
+        this.isRollbackCountEnabled = NiFiProperties.getInstance().isRollbackCountEnabled();
+        this.isRollbackLogUnackFFEnabled = NiFiProperties.getInstance().isRollbackLogUnackFFEnabled();
+        this.rollbackLogUnackFFMax = NiFiProperties.getInstance().getRollbackLogUnackFFMax();
+
```

Comment:
not about this line specifically but it doesn't look like we'd be resetting the flow file attributes after a given flow file was able to get passed a given processor's commit call.  Thus we could have rollback counts on attributes that don't make sense any more since it was about a different processor.  In either case, we won't have the attributes retained anyway because unless a commit happens there are no flow file attribute changes occurring.